src/ifcgeom/IfcIShapeProfileDef.cpp

```diff
@@ -65,8 +93,25 @@ bool IfcGeom::Kernel::convert(const IfcSchema::IfcIShapeProfileDef* l, TopoDS_Sh
 		IfcGeom::Kernel::convert(l->Position(), trsf2d);
 	}
 
-	double coords[24] = {-x1,-y, x1,-y, x1,-y+dy1, d1,-y+dy1, d1,y-dy2, x2,y-dy2, x2,y, -x2,y, -x2,y-dy2, -d1,y-dy2, -d1,-y+dy1, -x1,-y+dy1};
-	int fillets[4] = {3,4,9,10};
-	double radii[4] = {f1,f2,f2,f1};
-	return util::profile_helper(12,coords,(doFillet1||doFillet2) ? 4 : 0,fillets,radii,trsf2d,face);
+	double coords[24] = {
+		-x1,-y,
+		x1,-y,
+		x1,-y+ft1-dy2,
+		d1,-y+ft1+dy1,
+		d1,y-ft1-dy1,
+		x2,y-ft1+dy2,
+		x2,y,
+		-x2,y,
+		-x2,y-ft2+dy2,
+		-d1,y-ft2-dy1,
+		-d1,-y+ft2+dy1,
+		-x1,-y+ft2-dy2
+	};
+	int fillets[8] = {2,3,4,5,8,9,10,11};
+	double radii[8] = {fe1,f1,f2,fe2,fe2,f2,f1,fe1};
+#ifdef SCHEMA_IfcIShapeProfileDef_HAS_FlangeEdgeRadius
```

Comment:
It's a minor thing, but I'd prefer to have `doFlangeEdgeRadiu`s globally available and defaulting to false. That way this last ifdef is not necessary. That's the same approach we currently have with `doFillet2`.